extensions/wikia/Recirculation/services/ParselyDataService.class.php

```diff
@@ -105,12 +106,12 @@ private function buildUrl( $endpoint, $options ) {
 		$defaultParams = [
 			'apikey' => $wgParselyApiKey,
 			'secret' => $wgParselyApiSecret,
-			'page' => self::PARSELY_API_PAGE
+			'page' => static::PARSELY_API_PAGE
 		];
 
 		$params = array_merge( $defaultParams, $options );
 
-		$url = self::PARSELY_API_BASE . $endpoint . '?' . http_build_query( $params );
+		$url = static::PARSELY_API_BASE . $endpoint . '?' . http_build_query( $params );
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Immediately return this expression instead of assigning it to the temporary variable "$url". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1488)